kura/test/org.eclipse.kura.core.keystore.test/src/test/java/org/eclipse/kura/core/keystore/FilesystemKeystoreServiceImplTest.java

```diff
@@ -119,6 +121,43 @@ public void testGetKeyStore() throws KuraException, KeyStoreException {
         assertEquals(Collections.list(this.store.aliases()), Collections.list(keystore.aliases()));
     }
 
+    @Test
+    public void testCheckKeystoreFileCreation() throws KuraException, KeyStoreException {
+        Map<String, Object> properties = new HashMap<>();
```

Comment:
I would also check that the file we need to create doesn't exists before attempting its creation to avoid surprises

```suggestion
        assertFalse(new File(NEW_STORE_PATH).isFile());
        Map<String, Object> properties = new HashMap<>();
```